### PR TITLE
feat: add samples

### PR DESCRIPTION
**Pull request by dizcology**
**DO NOT MERGE** until sample tests passing.

### CODE REVIEW

**Comment by dizcology** [comment]
@googlebot I consent.

**Comment by vinnysenthil** [comment]
@googlebot I consent.

**Review by vinnysenthil — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 import setuptools  # type: ignore
 
 name = "google-cloud-aiplatform"
-version = "0.3.0"
+version = "0.3.1"
```
Comment (busunkim96) [inline setup.py:24]: No need to bump this here - if you merge this as a `feat` you'll see a PR for `0.4.0`.

**Review by busunkim96 — APPROVED** [review]
Rubber stamping per discussion over email on how sample reviewer comments will be addressed.

**Comment by Ark-kun** [comment]
>     # The AI Platform services require regional API endpoints.
>     client_options = {"api_endpoint": api_endpoint}

Are regional API endpoints still required?

**Pull request merged**